README.md

```diff
@@ -181,27 +182,47 @@ will install all the requirements necessary for plenoptic to run.
 
 ## Jupyter
 
-The one additional thing you will want is to install
-[JupyterLab](https://jupyterlab.readthedocs.io/en/stable/),
-which we use for tutorial and example notebooks:
+If you wish to locally run the notebooks, you will need to install `jupyter` and
+`ipywidgets` (you can also run them in the cloud using
+[Binder](https://mybinder.org/v2/gh/LabForComputationalVision/plenoptic/master?filepath=examples)).
+There are two main ways of getting a local `jupyter` install` working with this
+package:
+
+1. Install jupyter in the same environment as `plenoptic`. If you followed the
+   [instructions above](#plenoptic) to create a `conda` environment named
+   `plenoptic`, do the following:
 
+``` sh
+conda activate plenoptic
+conda install -c conda-forge jupyterlab ipywidgets
 ```
-conda install -c conda-forge jupyterlab
+
+   This is easy but, if you have multiple conda environments and want to use
+   Jupyter notebooks in each of them, it will take up a lot of space.
+   
+2. Use
+   [nb_conda_kernels](https://github.com/Anaconda-Platform/nb_conda_kernels).
+   Again, if you followed the instructions above:
+
+``` sh
+# activate your 'base' environment, the default one created by miniconda
+conda activate 
+# install jupyter lab and nb_conda_kernels in your base environment
+conda install -c conda-forge jupyterlab ipywidgets
```

Comment:
funnily enough I just had to do this to download the fashionMNIST dataset inside the Demo_Steerpyr notebook

Reply:
Yeah, I think pytorch / torchvision uses the notebook `tqdm` (and thus requires `ipywidgets`) for all their downloads